app/tests/spec/views/permissions.js

```diff
@@ -44,8 +41,7 @@ function (chai, $, sinon, p, View, Metrics, FxaClient, EphemeralMessages,
     var PERMISSIONS = ['profile:email', 'profile:uid'];
 
     beforeEach(function () {
-      email = TestHelpers.createEmail();
-
+      TestHelpers.createEmail();
```

Comment:
@vladikoff Just spoke with @ckarlof and we think that we should go ahead with the generated 'email' rather than the hardcoded 'EMAIL', does that sound good?